daemons/fenced/fenced_remote.c

```diff
@@ -1064,9 +1069,13 @@ fenced_handle_manual_confirmation(pcmk__client_t *client, xmlNode *msg)
     set_fencing_completed(op);
     op->delegate = strdup("a human");
 
-    // For the fencer's purposes, the fencing operation is done
+    {
+        // For the fencer's purposes, the fencing operation is done
+        pcmk__action_result_t result = PCMK__UNKNOWN_RESULT;
```

Comment:
for cases like this we might consider something like PCMK_RESULT_OK or even pcmk__result_ok.

Reply:
I considered that, but it could proliferate pretty easily ... having just one initializer makes it easy to base new code on any existing example. Alternatively we could make it like PCMK__INIT_RESULT(CRM_EX_OK, PCMK_EXEC_UNKNOWN) but that's harder to read